src/CrudPanel.php

```diff
@@ -226,4 +226,27 @@ public function sort($items)
 
         return $this->{$items};
     }
+
+    /**
+     * Get the Eloquent Model name from the given relation definition string.
+     *
+     * @example For a given string 'company' and a relation between App/Models/User and App/Models/Company, defined by a
+     *          company() method on the user model, the 'App/Models/Company' string will be returned.
+     *
+     * @example For a given string 'company.address' and a relation between App/Models/User, App/Models/Company and
+     *          App/Models/Address defined by a company() method on the user model and an address() method on the
+     *          company model, the 'App/Models/Address' string will be returned.
+     *
+     * @param $relationString String Relation string. A dot notation can be used to chain multiple relations.
+     *
+     * @return string relation model name
+     */
+    private function getRelationModel($relationString)
+    {
+        $result = array_reduce(explode('.', $relationString), function ($obj, $method) {
+            return $obj->$method()->getRelated();
```

Comment:
@tabacitu, since this error appears due to incorrect configuration by a programmer, I think we should try to provide as much detail as possible and, for our use case, the abort method lacks the full stack trace. I personally think that method should not be used inside frameworks and libraries, at it is intended to display human readable error messages to clients.

Regarding custom errors, I fully agree. Laravel does it, where it makes sense. In the situation of a misconfiguration, a BadMethodCallException with "Call to undefined method XYZ" would be thrown, which I think fits us perfectly.

If we were to do it differently, we should create something like a Backpack\Base\ConfigurationException class and use that. But we need to be consistent through all the packages and I think we should do that as part of another feature branch.

What do you guys think about this?